### PR TITLE
Data upload

### PR DESCRIPTION
**Pull request by sanjeevghimire**
* added steps to upload source data
* added steps to create tables that are organized by row
* readme changes
* flow diagram changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ When the reader has completed this Code Pattern, they will understand how to:
 2. The source data is then loaded, as data asset, into Watson Studio.
 3. The Watson Machine Learning service uses the source data and computes an evaluation using Apache Spark-as-a-service to create a machine learning model, and saves the evaluation information back to the Db2 Warehouse on Cloud database.
 4. Apache Spark-as-a-service to compute the evaluation.
-5. Feedback data is uploaded to Watson machine learning service to continuously learn and evaluate the new data.
+5. Feedback data is uploaded to the feedback table in DB2 warehouse.
```
Comment (rhagarty) [inline README.md:21]: you should be consistent - is it  "DB2", or "Db2" (2 lines up)

File: README.md [hunk 2]
```diff
@@ -49,14 +49,15 @@ The setup is done in 3 primary steps.  You will download the code, setup the app
...
-3. [Add and Refine data asset into Watson Studio](#3-add-and-refine-data-asset-into-watson-studio)
-4. [Create a Db2 Warehouse on Cloud database and add the connection to Watson Studio](#4-create-db2-warehouse-on-cloud-and-add-the-connection-to-watson-studio)
-5. [Create Apache Spark as a service with IBM Cloud](#5-create-apache-spark-as-a-service-with-ibm-cloud)
-6. [Create Watson Machine Learning with IBM Cloud](#6-create-watson-machine-learning-with-ibm-cloud)
-7. [Add new Watson Machine Learning Model to Watson Studio](#7-add-new-watson-machine-learning-model-to-watson-studio)
-8. [Add Feedback data and new evaluations to the continuously learning model](#8-add-feedback-data-and-new-evaluations-to-the-continuously-learning-model)
-9. [Deploy the model to expose it through an API](#9-deploy-the-model-to-expose-it-through-an-api)
-10. [Test the model](#10-test-the-model)
+3. [Create DB2 warehouse on cloud and add the connection to Watson Studio](#3-create-db2-warehouse-on-cloud-and-add-the-connection-to-watson-studio)
+4. [Create and load data to DB2 warehouse](#4-create-and-load-data-to-db2-warehouse)
```
Comment (rhagarty) [inline README.md]: think "into" would be a better term here

**Review by rhagarty — CHANGES_REQUESTED** [review]

**Comment by sanjeevghimire** [comment]
done

**Review by rhagarty — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (roger-e-sanders) [reply README.md:21]: It is Db2. You should also refer to the product as "Db2 Warehouse on Cloud". Db2 Warehouse is a completely different product.

**Pull request merged**